cmd/glasskube/cmd/install.go

```diff
@@ -61,6 +62,34 @@ var installCmd = &cobra.Command{
 			cancel()
 		}
 
+		// if --no-wait is used
+		if installCmdOptions.NoWait {
+			go func() {
+				status, err := install.NewInstaller(client).
+					WithStatusWriter(statuswriter.Spinner()).
+					InstallBlocking(ctx, packageName, installCmdOptions.Version)
```

Comment:
The installer instantiation can be lifted out of the "if" statement to reuse some code. 

Something like this:
```go
installer := install.NewInstaller(client).WithStatusWriter(statuswriter.Spinner())
if installCmdOptions.NoWait {
    ...
} else {
    ...
```